seldon/base/seldon-core-seldon-cluster-manager-deploy.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: seldon-core-seldon-cluster-manager
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: seldon-cluster-manager-server
+      app.kubernetes.io/component: seldon-core-operator
+      app.kubernetes.io/name: seldon-core
+      component: seldon-core
+  template:
+    metadata:
+      labels:
+        app: seldon-cluster-manager-server
+        app.kubernetes.io/component: seldon-core-operator
+        app.kubernetes.io/name: seldon-core
+        component: seldon-core
```

Comment:
you can remove `matchLabels` and `labels` here since `commonLabels` has done it

Reply:
done